apps/dashboard/src/utils/parseStepVariables.ts

```diff
@@ -150,7 +150,9 @@ export function parseStepVariables(schema: JSONSchemaDefinition): ParsedVariable
 
   return {
     ...result,
-    variables: [...result.primitives, ...result.namespaces],
+    variables: isEnhancedDigestEnabled
```

Comment:
This function is used by the hook that generates variables for all the fields with variables dropdown